packages/SwingSet/test/device-plugin/test-device.js

```diff
@@ -22,12 +21,15 @@ async function setupVatController(t) {
     inputQueue.push(thunk);
   };
 
-  const pluginRequire = mod => {
+  const importPlugin = mod => {
     t.is(mod, 'pingpong');
-    // eslint-disable-next-line global-require
-    return require('./pingpong');
+    return import('./pingpong.js');
```

Comment:
The need to replace `require` with async `import` here motivates much of the work above to make the plugin device have an async `createConnection` above.